tests/unit-tests/api/utilities/Tests_Functions.php

```diff
@@ -0,0 +1,497 @@
+<?php
```

Comment:
This file isn't 100% WPCS compliant. Whitelisting comments may be used where need be (example `echo` may be tagged with `// WPCS: XSS ok.`